wordpress-mobile/WordPress-Android
# Jetpack: SSO security settings

This is the last of the security settings for Jetpack. Note that there will be at least two subsequent PR's to fix some issues with the branch before merging into `develop`. I'll outline the remaining work in the ticket (#6194).

This PR adds functionality to the final section of [this screen](https://cldup.com/Wu-h_dhFjj.png), with one major exception being that `Match accounts using email addresses` changes cannot be saved remotely. This is one of the future PR's 😉  I'm working with the API folks to figure out where the option is exposed.

To test:
* Login to Jetpack site, login on the web to help test
* Make sure the initial values are correct
* Toggle the settings (save the second one) and make sure they're updated remotely

cc @oguzkocer
